packages/generator/src/generator-utils.ts

```diff
@@ -134,8 +134,15 @@ export function endWithDot(text: string): string {
   return !text || text.endsWith('.') || text.endsWith(':') ? text : `${text}.`;
 }
 
-export function linkClass(navProperty: VdmNavigationProperty): string {
-  return navProperty.isMultiLink ? 'Link' : 'OneToOneLink';
+export function linkClass(
+  navProperty: VdmNavigationProperty,
+  service: VdmServiceMetadata
+): string {
+  return navProperty.isCollection
```

Comment:
also here only pass the odata version

Reply:
Already done. Make sure to look at "Changes from all commits". This is a little confusing on GH.